libkbfs/backpressure_disk_limiter.go

```diff
@@ -14,42 +16,81 @@ import (
 )
 
 // backpressureDiskLimiter is an implementation of diskLimiter that
-// uses backpressure.
+// uses backpressure to slow down block puts before they hit the disk
+// limit.
+//
+// Let J be the (approximate) byte usage of the journal and F be the
+// free bytes on disk. Then we want to enforce
+//
+//   J <= min(J+F, L),
```

Comment:
One thing I forgot to do was to limit to a *fraction* of the available disk space (i.e., 25%) -- here we're limiting to 100% of the available disk space. But I think this is still reviewable.